src/js/Field.js

```diff
@@ -128,6 +128,14 @@
                 this.showingDefaultData = true;
             }
 
+            if(!Alpaca.isValEmpty(this.data) &&
+                !Alpaca.isEmpty(this.options["prefix"]) &&
+                !Alpaca.startsWith(this.data, this.options["prefix"])){
+                this.data = `${this.options["prefix"]}${this.data}`
+            }
+
+
+
```

Comment:
I think there are too many blank newlines here, I would just like to have one blank line at the top and bottom:

```
...

            if(!Alpaca.isValEmpty(this.data) &&
                !Alpaca.isEmpty(this.options["prefix"]) &&
                !Alpaca.startsWith(this.data, this.options["prefix"])){
                this.data = `${this.options["prefix"]}${this.data}`
            }

...
```